src/server/auth/lib/user-manager.js

```diff
@@ -553,6 +553,9 @@ router.get('/webida/api/oauth/myinfo',
         var user = req.user;
         delete user.passwordDigest;
         delete user.activationKey;
+        if (user.email.indexOf(conf.guestMode.accountPrefix) == 0) {
```

Comment:
Use '==='.
And it can be written in shorter form like `user.isGuest = (user.email.indexOf(conf.guestMode.accountPrefix) === 0);`